RichClipboard.package/RCClippingsWindow.class/instance/toggleAlphabeticSort.st

```diff
@@ -1,11 +1,16 @@
 sort-alphabetically
 toggleAlphabeticSort
+
 	self alphabeticSortInverted ifTrue: [ 
 		alphabeticSortInverted := false.
-		self clippingsList sortByName
+		self clippingsList sortByName.
+		self currentSortMode: ' (a-z)'.
+		self updateWindowTitle.
 	] ifFalse: [
 		alphabeticSortInverted := true.
-		self clippingsList sortByNameInverse
+		self clippingsList sortByNameInverse. 
+		self currentSortMode: ' (z-a)'.
+		self updateWindowTitle.
 	].
 	self changed: #alphabeticSortButtonLabel.
-	self fixHeightOfToolbarButtons
\ No newline at end of file
+	self fixHeightOfToolbarButtons
```

Comment:
Das "self updateWindowTitle" kann unten aufgerufen werden (und muss nicht in beiden Fallunterscheidungen einzeln hingeschrieben werden)

Reply:
Nachher die hinteren Punkte inn den Klammern noch entfernen